test_functionality/test_schulze_method.py

```diff
@@ -169,7 +169,10 @@ def test_tuple_ballots(self):
         output_list = SchulzeMethod(input_list, ballot_notation=SchulzeMethod.BALLOT_NOTATION_GROUPING).as_dict()
 
         # Run tests
```

Comment:
This appears to be unrelated to the rest of your PR, and should be part of a separate patch.